From cd55b98a178f8db27c48601493c760fdda005c20 Mon Sep 17 00:00:00 2001
From: Mario Domenech Goulart <mario@ossystems.com.br>
Date: Tue, 28 Feb 2012 17:56:33 -0300
Subject: [PATCH] prefs.js: disable shortcut for "browse with caret"

This setting disables the popup window that shows up when F7 is pressed.
---
 defaults/preferences/prefs.js |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/defaults/preferences/prefs.js b/defaults/preferences/prefs.js
index 3b362b4..c2c9d42 100644
--- a/defaults/preferences/prefs.js
+++ b/defaults/preferences/prefs.js
@@ -62,3 +62,6 @@ pref("privacy.sanitize.sanitizeOnShutdown", true);
 // Prevent slow script dialogs
 pref("dom.max_chrome_script_run_time", 0);
 pref("dom.max_script_run_time", 0);
+
+// Disable the popup window that shows up when F7 is pressed
+pref("accessibility.browsewithcaret_shortcut.enabled", false);
\ No newline at end of file
-- 
1.7.2.5

